templates/course_homepage/course_homepage.html

```diff
@@ -0,0 +1,37 @@
+<!DOCTYPE html>
+<html>
+  <head>
+     <meta charset="utf-8">
+      <title>NTU Course </title>
+      <link rel="stylesheet" href="course_homepage.css">
+      <script defer src="https://use.fontawesome.com/releases/v5.0.6/js/all.js"></script>
+   </head>
+
+   <!--大標-->
+   <body>
+      <div class="content">
```

Comment:
Try to use semantic HTML elements such as `<article>`, `<section>`, `<main>`, etc.
FYR: https://www.w3schools.com/html/html5_semantic_elements.asp